bpf/bpf_host.c

```diff
@@ -784,7 +784,7 @@ do_netdev_encrypt_fib(struct __ctx_buff *ctx __maybe_unused,
 	 * to let the stack decide how to egress the packet. EKS is the
 	 * example of an environment with multiple egress interfaces.
 	 */
-#if defined(BPF_HAVE_FIB_LOOKUP) && defined(ENCRYPT_IFACE)
```

Comment:
Out of curiosity, was there a reason for this change (2nd commit)? I think it's fine because we still have the `HAVE_` prefix to tell the reader that this is based on a kernel feature probe, but wondering if it was causing any issues.

Reply:
It wasn't causing any issues, more or less just a cosmetic change to have the same naming scheme for all feature probe macros.